samples/bluetooth/broadcast_assistant/README.rst

```diff
@@ -0,0 +1,50 @@
+.. zephyr:code-sample:: bluetooth_broadcast_assistant
+   :name: Bluetooth: Broadcast Assistant
+   :relevant-api: bt_bap
+
+   Bluetooth: Broadcast Assistant
+
+Overview
+********
+
+Application demonstrating the LE Audio broadcast assistant functionality.
+
+The sample will automatically try to connect to the Broadcast Audio Scan Service (BASS)
+on a preconfigured broadcast sink.  It will then search for a specific broadcast source and
+(if found) add the broadcast ID to the BASS.
+
+This sample can be found under
+:zephyr_file:`samples/bluetooth/broadcast_assistant` in the Zephyr tree.
+
+Check the :ref:`bluetooth samples section <bluetooth-samples>` for general information.
+
+Requirements
+************
+
+* BlueZ running on the host, or
+* A board with Bluetooth Low Energy 5.2 support
+
+Building and Running
+********************
+
+The application will act as a broadcast assistant with optionally preconfigured
+filtering of broadcast sink and broadcast source names. By default, the application will
+search for and connect to the first broadcast audio sink found (advertising PACS and
+BASS UUIDs) and then search for and select the first broadcast audio source found
+(advertising a broadcast ID).
+
+Filter these by modifying the following configs:
+
+``CONFIG_SELECT_SINK_NAME``: Substring of BT name of the sink.
+
+and
+
+``CONFIG_SELECT_SOURCE_NAME``: Substring of BT name or broadcast name of the source.
+
+Building for an nrf52840dk
+--------------------------
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/bluetooth/broadcast_assistant/
+   :board: nrf52840dk_nrf52840
+   :goals: build
```

Comment:
as we don't need the extra support for actual ISO streams and because the sample is not power hungry enough to require an nRF5340, I decided it was not needed and also decided to use the nRF52840 as build example. If someone wants to run it on an nRF5340 device, the readme has a link to the bluetooth samples overview with the info on how to build hci_ipc.

Reply:
Nonetheless, we still probably want to build this for the nRF5340 ADK in CI, given that it's audio related, right? And the above sample also adds instructions on how to build it for that.

Reply:
I can add instructions for how to build for nRF5340 Audio DK, but I'd prefer to add sysbuild & multiple targets in sample.yml in a subsequent PR, if that would be ok?